docs/loads/structuralpointdeformation.md

```diff
@@ -0,0 +1,25 @@
+# StructuralPointDeformation
+
+```{warning}
+New object under construction.
+```
+
+**Imposed deformation of a support**
+
+Displacement or rotation can be imposed on a support. Displacement can be set in the direction of one of the global axes, positive displacement acts in the positive direction of the corresponding axis. Rotation can be imposed around one of the global axes, positive rotation acts right-handed about the corresponding positive axis. A rigid support is required in the direction of the deformation.
+
+```{image} ../.gitbook/assets/41b_deformation_of_support.png
+:width: 600px
+```
+
+## Specification in the excel
+
+| Column header | Data type | Example / enum definition | Required | Description |
+|---|---|---|---|---|
+| Name | String | RS1 | yes | Human readable unique name of the deformation |
+| Support in a node | String | Sn6 | yes | The name of a support in a node on which the deformation load is applied |
```

Comment:
I'd either call this "Node" or "Reference node", to keep it consistent (e.g StructuralPointSupport / StructuralPointAction)

Reply:
Thank you for pointing this out. This comes to what do we want to assign it to and what is the direction we will expand it next. Will we add vector to the direction of the translation/rotation? Will we create LCS for support and use local XYZ of the support?